Laba3.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdlib.h>
+// For some reason Visual Studio wants scanf to be changed to scanf_s
+#define scanf scanf_s
+
+int main()
```

Comment:
Suggestion to divide code into sections with spaces between tasks, for clarity of reading